actions/debootstrap_action.go

```diff
@@ -79,12 +88,19 @@ func (d *DebootstrapAction) RunSecondStage(context debos.DebosContext) error {
 
 func (d *DebootstrapAction) Run(context *debos.DebosContext) error {
 	d.LogStart()
-	cmdline := []string{"debootstrap", "--no-check-gpg"}
+	cmdline := []string{"debootstrap"}
 
 	if d.MergedUsr {
 		cmdline = append(cmdline, "--merged-usr")
 	}
 
+	if !d.CheckGpg {
+		cmdline = append(cmdline, fmt.Sprintf("--no-check-gpg"))
+	} else if d.KeyringFile != "" {
```

Comment:
for extra brownie points; check in the verification these optoins aren't set such that they're in conflict

Reply:
I made `check-gpg: false` take precedence over `keyring-file` on the basis that it's still true that there is a "correct" keyring file, you're just not checking against it this time. But if you think that's too much of a trap I can make it fail validation.